examples/aws_ebs/aws-ebs-web.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: v1beta3
```

Comment:
Can you make this v1? We are in the middle (I think almost complete) of converting all of our examples to v1.

#9286